src/KubeLibrary/KubeLibrary.py

```diff
@@ -641,6 +641,28 @@ def get_daemonset_details_in_namespace(self, name, namespace):
         ret = self.appsv1.read_namespaced_daemon_set(name, namespace)
         return ret
 
+    def get_role_in_namespace(self, namespace):
```

Comment:
Better to call it *get_roles_in_namespace* as the returning value is list. Same for role binding**s**

Reply:
Done.